sentry-android-core/src/main/java/io/sentry/android/core/DefaultAndroidEventProcessor.java

```diff
@@ -880,34 +881,35 @@ private void setAppPackageInfo(final @NotNull App app, final @NotNull PackageInf
   }
 
   private @Nullable String[] getProGuardUuids() {
-    AssetManager assets = context.getAssets();
-    try {
-      String[] files = assets.list("");
-
-      List<String> listFiles = Arrays.asList(files != null ? files : new String[0]);
-      if (listFiles.contains("sentry-debug-meta.properties")) {
-        try (InputStream is = assets.open("sentry-debug-meta.properties")) {
-          Properties properties = new Properties();
-          properties.load(is);
-
-          String uuid = properties.getProperty("io.sentry.ProguardUuids");
-          if (uuid != null && !uuid.isEmpty()) {
-            return uuid.split("\\|", -1);
-          }
-          options
-              .getLogger()
-              .log(SentryLevel.INFO, "io.sentry.ProguardUuids property was not found.");
-        } catch (IOException e) {
-          options.getLogger().log(SentryLevel.ERROR, "Error getting Proguard UUIDs.", e);
+    final AssetManager assets = context.getAssets();
+
+    try (final InputStream is =
+        new BufferedInputStream(assets.open("sentry-debug-meta.properties"))) {
+      final Properties properties = new Properties();
+      properties.load(is);
+
+      final String uuid = properties.getProperty("io.sentry.ProguardUuids");
+      if (uuid != null && !uuid.isEmpty()) {
+        final String[] proguardUuids = uuid.split("\\|", -1);
+
+        // it should be only 1 proguard uuid, but the API accepts an Array so keeping it consistency
+        for (final String item : proguardUuids) {
+          options.getLogger().log(SentryLevel.DEBUG, "ProguardUuids found: %s", item);
```

Comment:
`l`: I think it is only one UUID.
```suggestion
          options.getLogger().log(SentryLevel.DEBUG, "ProguardUuid found: %s", item);
```